pkg/app/server.go

```diff
@@ -53,6 +53,7 @@ import (
 	"k8s.io/kube-state-metrics/v2/pkg/metricshandler"
 	"k8s.io/kube-state-metrics/v2/pkg/optin"
 	"k8s.io/kube-state-metrics/v2/pkg/options"
+	"k8s.io/kube-state-metrics/v2/pkg/util/collections"
```

Comment:
Can we use [sets](https://pkg.go.dev/k8s.io/apimachinery/pkg/util/sets) here instead?

Reply:
@murphd40 any progress? :)